model/github/github_repository.rb

```diff
@@ -1,9 +1,9 @@
 # frozen_string_literal: true
 
-require "aws-sdk-s3"
-
 require_relative "../../model"
 
+require "aws-sdk-s3"
+
 class GithubRepository < Sequel::Model
```

Comment:
extra spurious change?

Reply:
We generally require gems after requiring relative paths in our codebase. I just make it similar to others.

Reply:
ah okay, I'm going to be a bit of a pedant here, but it does make the patch confusing, because the subject is dealing with side effects of loading a library, so that load order may be implicated in something is not completely off-base. Although I'm not as pedantic about PRs glomming in a few unrelated changes, I would have understood that if it were committed separately as a cosmetic fix

Reply:
Fair point